src/Xamarin.Android.Build.Tasks/Tasks/AndroidUpdateResDir.cs

```diff
@@ -61,9 +61,9 @@ public class AndroidComputeResPaths : AndroidTask
 
 		public override bool RunTask ()
 		{
-			var intermediateFiles = new List<ITaskItem> ();
-			var resolvedFiles = new List<ITaskItem> ();
-			
+			var intermediateFiles = new Dictionary<string, ITaskItem> (ResourceFiles.Length, StringComparer.Ordinal);
```

Comment:
Filename comparisons generally use `StringComparer.OrdinalIgnoreCase`, because macOS and Windows are both case-insensitive (by default).